DisCatSharp/Entities/Application/DiscordApplicationCommandLocalization.cs

```diff
@@ -0,0 +1,107 @@
+// This file is part of the DisCatSharp project, a fork of DSharpPlus.
+//
+// Copyright (c) 2021 AITSYS
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+using System.Collections.Generic;
+using System.Linq;
+using System;
+using System.Text;
+
+namespace DisCatSharp.Entities
+{
+    /// <summary>
+    /// Represents a application command localization.
+    /// </summary>
+    public sealed class DiscordApplicationCommandLocalization
+    {
+        /// <summary>
+        /// Gets the localization dict.
+        /// </summary>
+        public Dictionary<string, string> Localizations { get; internal set; }
+
+        /// <summary>
+        /// Gets valid locales for Discord.
+        /// </summary>
+        internal List<string> _validLocales = new() { "ru", "fi", "hr", "de", "hu", "sv-SE", "cs", "fr", "it", "en-GB", "pt-BR", "ja", "tr", "en-US", "es-ES", "uk", "hi", "th", "el", "no", "ro", "ko", "zh-TW", "vi", "zh-CN", "pl", "bg", "da", "nl", "lt" };
```

Comment:
is it really a fixed, predetermined list? just wondering

Reply:
Yes. Discord uses its own system.
The API only accept these.
See https://docs.dcs.aitsys.dev/localization/articles/application_commands/translations/reference.html#valid-locales